과제/week4/src/pages/loginPage.tsx

```diff
@@ -0,0 +1,9 @@
+import Login from "../components/Login";
+
+export default function LoginPage() {
+  return (
+    <>
+      <Login />
+    </>
```

Comment:
Login컴포넌트를 호출만 할거면 왜 page컴포넌트를 만들었나요??

Reply:
```suggestion
    <Login />
```
추가로 이런경우에는 <> </>로 감쌀 필요 없을 것 같습니다!